debug_admin_role.py

```
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+"""Debug admin user role and credit account creation"""
+
+import requests
+import json
+
+BASE_URL = "https://cryptouniverse.onrender.com/api/v1"
+ADMIN_EMAIL = "admin@cryptouniverse.com"
+ADMIN_PASSWORD = "AdminPass123!"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "ADMIN_PASSWORD"

(S105)